draft-ietf-quic-transport.md

```diff
@@ -943,47 +940,104 @@ The fields in the ACK frame are as follows:
+  indicates the number of contiguous packets being acked starting after the
+  end of the previous gap.  Along with the previous field, this field is
+  repeated "Num Blocks" times.
 
-  * Ack Block Length (opt, repeated): An unsigned packet number delta that
-    indicates the number of contiguous packets being acked starting after the
-    end of the previous gap.  Along with the previous field, this field is
-    repeated "Num Blocks" times.
+### Timestamp Section {#timestamp-section}
+
+The Timestamp Section contains between zero and 255 measurements of packet
+delay. This information can be used by a sender to better estimate the RTT of
+the connection.
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|   Num TS(8)   | [Delta LA (8)]|
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                [Time Since Largest Acked (32)]                |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|[Delta LA 1(8)]| [Time Since Previous 1 (16)]  |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|[Delta LA 2(8)]| [Time Since Previous 2 (16)]  |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+                             ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|[Delta LA N(8)]| [Time Since Previous N (16)]  |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+~~~
+{: #timestamp-format title="Timestamp Section"}
 
-* Timestamp Section:
+The fields in the Timestamp Section are:
 
-  * Num Timestamps: An unsigned 8-bit number specifying the total number of
-    <packet number, timestamp> pairs following, including the First Timestamp.
+* Num Timestamps: An unsigned 8-bit number specifying the total number of
+  <packet number, timestamp> pairs following, including the First Timestamp.
 
-  * Delta Largest Acked (opt): An optional 8-bit unsigned packet number delta
-    specifying the delta between the largest acked and the first packet whose
-    timestamp is being reported.  In other words, this first packet number may
-    be computed as (Largest Acked - Delta Largest Acked.)
+* Delta Largest Acked (opt): An optional 8-bit unsigned packet number delta
+  specifying the delta between the largest acked and the first packet whose
+  timestamp is being reported.  In other words, this first packet number may
+  be computed as (Largest Acked - Delta Largest Acked.)
 
-  * First Timestamp (opt): An optional 32-bit unsigned value specifying the time
-    delta in microseconds, from the beginning of the connection to the arrival
-    of this packet.
+* First Timestamp (opt): An optional 32-bit unsigned value specifying the time
+  delta in microseconds, from the beginning of the connection to the arrival
+  of this packet.
 
-  * Delta Largest Observed (opt, repeated): (Same as above.)
+* Delta Largest Acked 1-N (opt, repeated): (Same as above.)
```

Comment:
Maybe use an ellipsis here rather than the hyphen, which my addled brain interpreted as minus.